docs/contributing.rst

```diff
@@ -39,14 +39,26 @@ Accessing dev server
 In any case, you can point your browser at `http://localhost:5000 <http://localhost:5000>`_.
 It's as simple as that!
 
-
 Updating
 --------
 
 In case you want to update to newer versions (from git), you can just run the "update" command::
 
   make update
 
+Create database migrations
+--------------------------
+
+In case you need to modify the database schema, first update the models in ihatemoney/models.py.
+Then run the following command to create a new database revision file::
+
+  make create-database-revision
+
+If your changes are simple enough, the generated script will be populated with
+the necessary migrations steps.
+For complex migrations, these steps need to be written manually. Existing revision files
```

Comment:
Actually, this sentence about complex migrations is a bit vague, and IMHO reveals a slight lack in our tooling : a simple command to create empty migration. Current command to do that is :

```
python -m ihatemoney.manage db revision -d ihatemoney/migrations
```

(given that we are already in virtualenv, which is not an assumption we make for other dev-related commands).

Maybe it is good time to add a `make create-empty-database-revision` and adapt doc, WDYT ?

Reply:
+1 for the Makefile command.

Reply:
@0livd gentle bump :-). any time for this ?

Reply:
Yep, I've just added the new makefile command :)